h/models/user_identity.py

```diff
@@ -13,6 +13,9 @@ class UserIdentity(Base):
     id = sa.Column(sa.Integer(), autoincrement=True, primary_key=True)
     provider = sa.Column(sa.UnicodeText(), nullable=False)
     provider_unique_id = sa.Column(sa.UnicodeText(), nullable=False)
+    user_id = sa.Column(
+        sa.Integer(), sa.ForeignKey("user.id", ondelete="cascade"), nullable=False
```

Comment:
The `ondelete="cascade"` doesn't change anything about how our Python code behaves - all the tests would still pass without it. But it does cause sqlalchemy to add the `ON DELETE CASCADE` to the `user_identity` -> `user` foreign key relationship, which is useful for anyone working the DB directly in SQL.